File: Create_File/Create.py
"""
    Author : Pakshal Shashikant Jain 
    Date : 17/05/2024
    Program : Write application which accept file name from user and create that file.
              
              Input : Demo.txt
              Output : File created successfully. 
"""

##################################################################################################

#Function To Create File
def CreateFile(fname) :
    #Creating File
    fd = open(fname,'w',encoding = 'utf-8');

    if(fd == None) :
        print("Unable To Create File");
    else :
        print("File Successfully Created");

##################################################################################################

#Main Function
def main() :
    fname = " ";

    print("Jay Ganesh....");

    #Taking Input From User
    print("Enter File Name To Create");
    fname = input();

    #Call To CreatFile Function
    CreateFile(fname);

###################################################################################################

#Call Back Method To Call Main Function
if __name__ == "__main__" :
    #Call To Main Function
    main();

File: Create_File/Create.c
/*
    Author : Pakshal Shashikant Jain 
    Date : 17/05/2024
    Program : Write application which accept file name from user and create that file.
              
              Input : Demo.txt
              Output : File created successfully. 
*/

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Required Libraries
#include<stdio.h>
#include<stdlib.h>
#include<fcntl.h>
#include<unistd.h>

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Function To Create File
void CreateFile(char fname[]) 
{
    int fd = 0;

    //Creating File
    fd = creat(fname,0x777);

    if(fd == -1)
    {
        printf("Unable To Create File \n");
    }
    else 
    {
        printf("File Successfully Create With File Descriptor : %d",fd);
    }

    close(fd);
}

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Function
int main()
{
    char fname[40] = {'\0'};

    printf("Jay Ganesh....\n");

    //Taking File Name From User
    printf("Enter File Name You Want To Create: ");
    scanf("%[^'\n']s",&fname);

    //Call To CreateFile Function
    CreateFile(fname);

    return 0;
}